  @Test
  void test_get_no_root_cause() {
    Exception ex = new Exception();
    assertThat(ThrowableUtils.getRootCause(ex)).isEqualTo(ex);
  }

